src/veh_type.h

```diff
@@ -80,6 +80,7 @@ enum vpart_bitflags : int {
     VPFLAG_WASHING_MACHINE,
     VPFLAG_FLUIDTANK,
     VPFLAG_REACTOR,
+    VPFLAG_FLAT_SURF,
```

Comment:
quoting the header file:
```
// bitmask backing store of -certain- vpart_info.flags, ones that
// won't be going away, are involved in core functionality, and are checked frequently
enum vpart_bitflags : int {
```
this is not core functionality nor is it checked frequently.  do not put it in the vpart_bitflags.